### PR TITLE
Add new symbol issue for the case when the snupkg does not contain pdbs

### PR DESCRIPTION
**Pull request by cristinamanum**
Add new symbol issue for the case when the snupkg does not contain any symbols.

### CODE REVIEW

File: src/NuGet.Services.Validation.Issues/ValidationIssue.cs [hunk 1]
```diff
@@ -48,7 +49,8 @@ public abstract class ValidationIssue : IValidationIssue
             ValidationIssueCode.PackageIsNotSigned,
             ValidationIssueCode.SymbolErrorCode_ChecksumDoesNotMatch,
             ValidationIssueCode.SymbolErrorCode_MatchingAssemblyNotFound,
-            ValidationIssueCode.SymbolErrorCode_PdbIsNotPortable
+            ValidationIssueCode.SymbolErrorCode_PdbIsNotPortable,
+            ValidationIssueCode.SymbolErrorCode_SnupkgDoesNotContainSymbols
```
Comment (joelverhagen) [inline src/NuGet.Services.Validation.Issues/ValidationIssue.cs]: nit: trailing comma to reduce future diff

File: tests/NuGet.Services.Validation.Issues.Tests/ValidationIssuesFacts.cs [hunk 2]
```diff
@@ -242,7 +242,8 @@ private PackageValidationIssue CreatePackageValidationIssue(ValidationIssueCode
             { ValidationIssueCode.PackageIsNotSigned, () => ValidationIssue.PackageIsNotSigned },
             { ValidationIssueCode.SymbolErrorCode_ChecksumDoesNotMatch, () => ValidationIssue.SymbolErrorCode_ChecksumDoesNotMatch },
             { ValidationIssueCode.SymbolErrorCode_MatchingAssemblyNotFound, () => ValidationIssue.SymbolErrorCode_MatchingAssemblyNotFound },
-            { ValidationIssueCode.SymbolErrorCode_PdbIsNotPortable, () => ValidationIssue.SymbolErrorCode_PdbIsNotPortable }
+            { ValidationIssueCode.SymbolErrorCode_PdbIsNotPortable, () => ValidationIssue.SymbolErrorCode_PdbIsNotPortable },
```
Comment (joelverhagen) [inline tests/NuGet.Services.Validation.Issues.Tests/ValidationIssuesFacts.cs]: nit: trailing comma to reduce future diff

Comment (cristinamanum) [reply tests/NuGet.Services.Validation.Issues.Tests/ValidationIssuesFacts.cs]: Updated

File: src/NuGet.Services.Validation.Issues/ValidationIssue.cs (hunk 1 above)
Comment (cristinamanum) [reply src/NuGet.Services.Validation.Issues/ValidationIssue.cs]: Updated

**Pull request merged**